tests/test_celery_unique.py

```diff
@@ -4,16 +4,15 @@
 
 import datetime
 import inspect
-import pytz
 import unittest
 from uuid import uuid4
 
-from celery.result import AsyncResult
 import mock
```

Comment:
Since we're moving this to be compatible with Python 3, I think we should prefer using `unittest.mock` on versions of Python where it's available, instead of `import mock`, which will always rely on the contrib `mock` package:
```python
try:
    from unittest import mock
except ImportError:
    import mock
```